scripts/generate_client_publish_job.py

```diff
@@ -23,12 +23,20 @@
 
 def initWorkspace():
     path = tempfile.mkdtemp()
-    subprocess.check_output(
+    subprocess.check_call(
         ["git", "clone", "https://github.com/mendersoftware/integration.git", path],
     )
     return path
 
 
+def getDefaultMetaMender():
+    path = tempfile.mkdtemp()
+    subprocess.check_call(
+        ["git", "clone", "https://github.com/mendersoftware/mender-qa.git", path],
```

Comment:
I was thinking for both scripts that you don't need this, because this is the mender-qa repository. But I suppose it is because this script is imported? 

Using `requests.get` or `wget` like the other script will be considerably more efficient, but I'll leave it to you if you want to bother or not.

Reply:
> I was thinking for both scripts that you don't need this, because this is the mender-qa repository. But I suppose it is because this script is imported?
> 

Correct, it is imported from different pipelines

> Using `requests.get` or `wget` like the other script will be considerably more efficient, but I'll leave it to you if you want to bother or not.

I thought about `wget` or `requests` but discarded it as I didn't want to introduce a new dependency (and then modify all repos pipelines and their release branches).

However, I could use `urllib` which is part of the standard library :thinking:

Reply:
Done